test/unit/gateways/worldpay_test.rb

```diff
@@ -1588,6 +1600,22 @@ def test_authorize_prefers_options_for_ntid
     assert_success response
   end
 
+  def test_authorize_recurring_apple_pay_with_ntid
+    stored_credential_params = stored_credential(:used, :recurring, :merchant, network_transaction_id: '3812908490218390214124')
+
+    @options.merge({
+      stored_credential: stored_credential_params,
+      stored_credential_transaction_id: '000000000000020005060720116005060',
+      wallet_type: :apple_pay
+    })
+    response = stub_comms do
+      @gateway.authorize(@amount, @apple_pay_network_token_without_payment_cryptogram, @options)
+    end.check_request do |_endpoint, data, _headers|
+      assert_match %r(<EMVCO_TOKEN-SSL type="APPLEPAY">), data
```

Comment:
Can we also add a check to ensure the cryptogram is not sent? Maybe something like this:
```suggestion
      refute_match %r(<cryptogram>), data
```